src/main/logutils.ts

```diff
@@ -117,9 +122,13 @@ const handleLogLine = (line: string) => {
     } else if (line.includes("ARENA_MATCH_END")) {
         handleArenaStopLine(line);
     } else if (line.includes("ENCOUNTER_START")) {
-        handleRaidStartLine(line);
+        handleEncounterStartLine(line);
     } else if (line.includes("ENCOUNTER_END")) {
-        handleRaidStopLine(line);
+        handleEncounterStopLine(line);
+    } else if (line.includes("CHALLENGE_MODE_START")) {
+        handleChallengeModeStartLine(line);
+    } else if (line.includes("CHALLENGE_MODE_END")) {
+        handleChallengeModeStopLine(line);
```

Comment:
if a group disbands, i dont think we get the challenge mode end event, right?

Reply:
No, I was planning on working on a combat log timeout for this before I sent it in a PR. Reading the comments I believe that could help for BGs too?